test/e2e/toolchaincluster_test.go

```diff
@@ -174,3 +176,86 @@ func caBundle(bundle string) clusterOption {
 		c.Spec.CABundle = bundle
 	}
 }
+
+func TestForceMetricsSynchronization(t *testing.T) {
+	// given
+	toolchainClusterList := &toolchainv1alpha1.ToolchainClusterList{}
+	ctx, hostAwait, memberAwait, member2Await := WaitForDeployments(t, toolchainClusterList)
+	defer ctx.Cleanup()
+	hostAwait.UpdateToolchainConfig(
+		testconfig.AutomaticApproval().Enabled(),
+		testconfig.Metrics().ForceSynchronization(false))
+	t.Cleanup(func() {
+		err := hostAwait.ScaleDeployment(hostAwait.Namespace, "host-operator", 1)
+		require.NoError(t, err)
+	})
+	// before creating a batch of users, let's remove all remainings of previous tests, so
+	// we know exactly what numbers we're dealing with
```

Comment:
wouldn't it be easier to list existing UserSignups and store the number at the beginning of the test?